index.js

```diff
@@ -39,4 +44,6 @@ clear.match = regex => {
 	}
 };
 
+clear.recursive = moduleId => clear(moduleId, true);
```

Comment:
I don't really see the point of having a separate method in addition to the argument on the default export.

Reply:
I agree. Just wanted to allow a clean call option. Can be removed